projects/1.20/assets/tome-of-blood-rebirth/ars_nouveau/patchouli_books/worn_notebook/zh_cn/entries/tomeofblood/mint_tea.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "薄荷茶",
+  "icon": "tomeofblood:mint_tea",
+  "category": "ars_nouveau:tomeofblood",
+  "sortnum": 2,
+  "pages": [
+    {
+      "type": "patchouli:spotlight",
+      "item": "tomeofblood:mint_tea",
+      "text": "$(o)带有微弱毒性……$()$(br2)喝下可获得$(9)宁静$()药水效果，可在血之祭坛自我献祭时额外获得10%%血液。$(br2)$(o)有5%%概率显现毒性。$()"
```

```suggestion
      "text": "$(o)带有微弱毒性……$()$(br2)喝下可获得$(9)宁静$()药水效果，可在血之祭坛自我献祭时额外获得 10%% 血液。$(br2)$(o)有 5%% 概率显现毒性。$()"
```
另请通查全角与半角符号之间的空格添加（为patchouli手册）

---

arsnouveau本体就没有空格来着

---

草了……那就和模组保持统一吧